openfecwebapp/config.py

```diff
@@ -12,7 +12,8 @@
 api_key_public = env.get_credential('FEC_WEB_API_KEY_PUBLIC', '')
 cache = os.getenv('FEC_WEB_CACHE')
 cache_size = int(os.getenv('FEC_WEB_CACHE_SIZE', 1000))
-
+transition_url = env.get_credential('FEC_TRANSITION_URL', 'http://www.fec.gov')
```

Comment:
I think the transition URL is stable enough now where you can change this variable to https://transition.fec.gov